mycroft/audio/audioservice.py

```diff
@@ -495,6 +501,38 @@ def _list_backends(self, message):
             data[s.name] = info
         self.bus.emit(message.response(data))
 
+    def _get_track_length(self, message):
+        """
+        getting the duration of the audio in milliseconds
+        """
+        dur = None
+        if self.current:
+            dur = self.current.get_track_length()
+        self.bus.emit(message.response({"length": dur}))
+
+    def _get_track_position(self, message):
+        """
+        get current position in milliseconds
+
+          Args:
+                seconds (int): number of seconds of final position
```

Comment:
could seconds be a float?
This is the expected key on the `message.data` object right?

Reply:
it could be a float, but vlc and probably most backend require an int representing milliseconds, so in case of a float it introduces imprecisions. as per comment above I think we maybe should use milliseconds which removes the uncertainty